### PR TITLE
updates deps and changelog

### PR DESCRIPTION
**Pull request by andrewpmartinez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +12,36 @@
   - N/A
 
 # Edge
+## Breaking Changes
+
+The following Edge Management REST API Endpoints have breaking changes:
```
Comment (dovholuknf) [inline CHANGELOG.md:17]: should this be 0.26.x?

Comment (andrewpmartinez) [reply CHANGELOG.md:17]: Reasonable question. Below v1.0.0 breaking changes have no sway over versioning.

Comment (plorenz) [reply CHANGELOG.md:17]: We've generally been trying to bump the minor to indicate breaking changes

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +12,36 @@
...
+- `PUT /ext-jwt-signers` - `kid` is required if `certPem` is specified, `issuer` is required, `audience` is required
+  - `kid` is required if `certPem` is specified
+  - `jwtEndpoint` or `certPem` is required
+  - `issuer` is now required
+  - `audience` is now required
+- `PATCH /ext-jwt-signers` - `kid` is required if `certPem` is specified, `issuer` is required, `audience` is required
+  - `kid` is required if `certPem` is set and `kid` was not previously set
+  - `jwtEndpoint` or `certPem` must be defined or previously set of the other is  `null`
+  - `issuer` may not be set to `null` or `""`
+  - `audience` may not be set to `null` or `""`
```
Comment (dovholuknf) [inline CHANGELOG.md:33]: nothing but whitespace is ok? on these last two fields that clearly specify no to null or ""?

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +12,36 @@
...
+
+The above changes will render existing `ext-jwt-signers` as always failing authentication is `issuer` and `audience`
```
Comment (dovholuknf) [inline CHANGELOG.md:35]: > failing authentication is

typo? should be 'if'?

**Pull request merged**